test/MOI_wrapper.jl

```diff
@@ -78,3 +78,35 @@ end
         "solve_objbound_edge_cases"
     ])
 end
+
+@testset "Test params" begin
+    # This test checks that options are passed through to the solver. We test
+    # with a time limit (seconds=0). To do so, we need to construct a MIP that
+    # is non-trivial to solve. We use a binary knapsack with unusual
+    # coefficients.
+    knapsack_model = ModelForCachingOptimizer{Float64}()
+    x = MOI.add_variables(knapsack_model, 100)
+    for i in 1:length(x)
+        MOI.add_constraint(knapsack_model, MOI.SingleVariable(x[i]), MOI.ZeroOne())
+        MOI.add_constraint(knapsack_model, MOI.SingleVariable(x[i]), MOI.GreaterThan(0.0))
```

Comment:
Isn't this redundant with `ZeroOne`?

Reply:
That might be a bug in the Cbc wrapper then. I think I tried and got unbounded.